src/Files.App/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -180,6 +181,12 @@ public override List<ContextMenuFlyoutItemViewModel> GetItemMenuItems(WidgetCard
 					ShowItem = options?.ShowFormatDrive ?? false
 				},
 				new ContextMenuFlyoutItemViewModel()
+				{
+					Text = "OpenStorageSense".GetLocalizedResource(),
+					Command = GoToStorageSenseCommand!,
+					CommandParameter = item
+				},
```

Comment:
Can you put this after properties?

Reply:
Properties shouldn't be always on the bottom??

Reply:
I don't understand what you mean.

Quick Access widget card item right clicked in Files
![image](https://github.com/files-community/Files/assets/62196528/bb36dc3f-376d-4517-9c6d-fb7984b42f8b)

Drives section right clicked in File Explorer
![image](https://github.com/files-community/Files/assets/62196528/5addd092-7536-4265-9d35-11ecde48302c)

Reply:
I'm not sure why File Explorer is like that

Reply:
I assume it's because Microsoft has always implemeneted as below.

![image](https://github.com/files-community/Files/assets/62196528/44483de9-00d7-48af-ad9d-920b65916d48)

Reply:
@mdtauk What do you think>

Reply:
> it's consistent with the Quick Access widget.

I don't understand; Quick Access widget specific items don't exist. Can you clarify what you mean?

> Can you put this after properties?

In my opinion I don't want to do so because it is always displayed on a context flyout of any item and should be on a specific place from UX aspects. Idk @mdtauk says but I don't without clear reason why.

Reply:
Items that are the same across all widgets should always be in the same position, this helps with muscle memory and improves the consistency throughout Files.

Reply:
Being at the bottom of couse helps with muscle memory. I can say definitely tho.

Reply:
I should have talk about this before work on this PR. I think I will add support to run RichCommand from widgets and sidebar and so in the mean time this work will be wasted/removed.

Could I close and discuss in the issue?